### PR TITLE
Invite volunteers event ids

### PR DESCRIPTION
**Pull request by karbmk**
Invite volunteers from 
1. create-event => (GET) invite-volunteers/[event_ids]
2. admin-profile => (GET) invite-volunteers/event_id (through event menu)
3. menu => (GET) invite-volunteers (through separate button)
built.

"event_ids" is the URL param and it will should always be a list while parsing in any view.

### CODE REVIEW

**Comment by nickolashe** [comment]
@karbmk it looks like there are more changes here than those related to the subject of the PR. did you branch off the new time tracker by any chance?

**Comment by karbmk** [comment]
I actually merged develop and the adminactionusertime branch

**Comment by nickolashe** [comment]
@karbmk yes, we have a solution for the missing table. @dannypernik or i can help. lets try to have only the changes related to the developed feature in the PR.

**Comment by dannypernik** [comment]
@nickolashe I'm removing links to invite-volunteers and edit-event in develop. Edit-event has multiple bugs ( #403 ) and we can add back the link to invite-volunteers when we merge this.

**Comment by dannypernik** [comment]
@nickolashe lots of conflicts in views.py here that I'm not comfortable resolving

**Comment by nickolashe** [comment]
@karbmk perhaps you can help resolve these?

**Comment by nickolashe** [comment]
@karbmk can we also make sure the PR contains only the changes pertaining to the fix

File: openCurrents/views.py [hunk 1]
```diff
@@ -434,6 +432,83 @@ def post(self, request, **kwargs):
         logger.info(
             'templist: %s', templist
         )
+        projects = []
+        userid = self.request.user.id
+        org = OrgUserInfo(userid)
+        orgid = org.get_org_id()
+        for i in templist:
```
Comment (nickolashe) [inline openCurrents/views.py]: @karbmk it seems like we are duplicating the code for parsing hour action request below (line 524)?!

File: openCurrents/views.py [hunk 2]
```diff
@@ -1228,9 +1315,14 @@ def form_valid(self, form):
         else:
             self.project = None
 
+        # public boolean - needs to handled through forms
+        if str(self.request.POST['is_public']) == '1':
```
Comment (nickolashe) [inline openCurrents/views.py]: @karbmk i think this is overwriting the changes i pushed to develop last night

Comment (karbmk) [reply openCurrents/views.py]: I will take a look on it

File: openCurrents/views.py (hunk 1 above)
Comment (karbmk) [reply openCurrents/views.py]: I think it was because of merging. I have it rectified now

File: openCurrents/views.py [hunk 3]
```diff
@@ -1514,6 +1511,11 @@ def get_context_data(self, **kwargs):
         context['skip'] = 0
         try:
             event_ids = kwargs.pop('event_ids')
+            if type(json.loads(event_ids)) == list:
+                pass
+            else:
+                event_ids = '['+event_ids+']'
```
Comment (nickolashe) [inline openCurrents/views.py]: @karbmk in essence, we are trying to encoding json on our own here - something should be avoided. lets build the (single element) list and encode it using json instead.

File: openCurrents/views.py [hunk 4]
```diff
@@ -1534,7 +1536,13 @@ def post(self, request, *args, **kwargs):
         post_data = self.request.POST
         event_create_id = None
         try:
-            event_create_id = json.loads(kwargs.pop('event_ids'))
+            event_create_id = kwargs.pop('event_ids')
+            if type(json.loads(event_create_id)) == list:
+                pass
+            else:
+                event_create_id = '['+event_create_id+']'
```
Comment (nickolashe) [inline openCurrents/views.py]: same comment as above

Comment (karbmk) [reply openCurrents/views.py]: @nickolashe , taken care of it added the element in the list and unicoded. check that once please. Also, tested the changes locally

**Pull request merged**